src/components/view/Navigation/index.js

```diff
@@ -12,19 +13,14 @@ import { blueGrey600, cyan600 } from 'material-ui/styles/colors'
 import ExpandMoreIcon from 'material-ui/svg-icons/navigation/expand-more';
 import AssignmentInd from 'material-ui/svg-icons/action/assignment-ind';
 import ChromeReaderModeIcon from 'material-ui/svg-icons/action/chrome-reader-mode';
-import CreditCardIcon from 'material-ui/svg-icons/action/credit-card';
-import RedeemIcon from 'material-ui/svg-icons/action/redeem';
-// import TimelineIcon from 'material-ui/svg-icons/action/timeline';
-import LocalPlayIcon from 'material-ui/svg-icons/maps/local-play';
-// import LocalOfferIcon from 'material-ui/svg-icons/maps/local-offer';
-// import MoodIcon from 'material-ui/svg-icons/social/mood';
-import PollIcon from 'material-ui/svg-icons/social/poll';
-// import PeopleIcon from 'material-ui/svg-icons/social/people';
+import PeopleIcon from 'material-ui/svg-icons/social/people';
 import PersonIcon from 'material-ui/svg-icons/social/person';
-// import WhatsHotIcon from 'material-ui/svg-icons/social/whatshot';
-// import StarIcon from 'material-ui/svg-icons/toggle/star';
-// import AddCircleIcon from 'material-ui/svg-icons/content/add-circle';
-import ModeEditIcon from 'material-ui/svg-icons/editor/mode-edit';
+import PersonOutlineIcon from 'material-ui/svg-icons/social/person-outline';
```

Comment:
Are you using all of these icons? If not, please comment out any that aren't used.

Thanks for following the icon naming pattern!

#recommended